workflow-templates/workflow-lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Workflow Lint
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+  schedule:
+    # Noon (EST) once a month
+    - cron: '0 4 1 * *'
```

Comment:
Why also have a schedule?
Plenty of repos won't habitually update their workflows. If we do this we can pull failed workflow runs from the API at some point.

Monthly might be a long time thought. Thoughts about daily?

E.g. say you add a new lint here, you'd need to wait until this fired again to see for sure which repos failed the lint.

Reply:
Monthly is a long time, but maybe daily is a lot?  Weekly?

Reply:
Might be hard with cron 😬

Reply:
The last `*` is day of the week: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07

Reply:
Ah, you got it

Reply:
https://slack.d2l.dev/emoji/doh